reconcile/status_board.py

```diff
@@ -240,7 +240,24 @@ def create_app(app_name: str, product: Product) -> Application:
     def apply_diff(
         dry_run: bool, ocm_api: OCMBaseClient, diff: list[StatusBoardHandler]
     ) -> None:
-        for d in sorted(diff, key=lambda x: x.status_board_object.get_priority()):
+
+        creations:list[StatusBoardHandler] = []
+        deletions:list[StatusBoardHandler] = []
+
+        for o in diff:
+            match o.action:
+                case "create":
+                    creations.append(o)
+                case "delete":
+                    deletions.append(o)
```

Comment:
That looks unnecessary, since you create the list via comprehension a little lower again

Reply:
oh sry, I changed to this to just iterate one time over the diffs